src/ProductBundle/Resources/config/command.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+	<services>
+		<service id="Sonata\ProductBundle\Command\GenerateProductCommand">
+			<tag name="console.command" command="sonata:product:generate"/>
```

Comment:
The command attribute isn necessary because it is already set. As @greg0ire mentioned it would be better to move `src/OrderBundle/Resources/config/command.xml` to `src/ProductBundle/Resources/config/command.xml`.
https://github.com/sonata-project/ecommerce/blob/299ad332952859988b297ff9adf2362be4e1de78/src/ProductBundle/Command/GenerateProductCommand.php#L50